Things.md

```diff
@@ -0,0 +1,181 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing. The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their properties, placements, etc.  
+  
+### Table of Contents
+1. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+    2. [Opacity](#opacity)
+2. [GroupHoldr](#groupholdr)
+    1. [Basic Architecture](#basic-architecture)
+        1. [Important APIs](#important-apis)
+        2. [Constructor Arguments](#constructor-arguments)
+    3. [Sample Usage](#sample-usage)
+
+# Manipulating Things
+
+## Shifting Things  
+A Thing's placement on the screen is based on its top, bottom, right, and left properties. These define the bounding box for a Thing. There are a number of ways to shift Things, all of which is done using EightBitter and by manipulating these 4 properties:  
```

Comment:
I like how use codified the `opacity` property below. Do that for left, etc.

Also, side note: the customary order in FSS is top/right/bottom/left (because that's what CSS uses).